presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -1274,26 +1279,65 @@ public ConnectorTableLayout getTableLayout(ConnectorSession session, ConnectorTa
             discretePredicates = Optional.of(new DiscretePredicates(partitionColumns, partitionDomains));
         }
 
+        Optional<ConnectorNodePartitioning> nodePartitioning = Optional.empty();
+        if (bucketExecutionEnabled && hiveLayoutHandle.getBucketHandle().isPresent()) {
```

Comment:
Should we introduce a session property as an escape hatch for broken tables?  Or maybe we should look for a table property that says, "this table has bad bucket information"